scripts/verify-prod-env.mjs

```
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+/**
+ * 프로덕션 배포 전 환경변수 안전성 검증.
+ * Vercel/Netlify 프로덕션 빌드 파이프라인에서 호출하여 실수로 mocking 이 켜지지 않도록 막는다.
+ */
+const errors = [];
+
+if (process.env.NEXT_PUBLIC_API_MOCKING === 'true') {
```

ESLint

[error] 'process' is not defined.

(no-undef)